ballerina-tests/Dependencies.toml

```diff
@@ -33,7 +33,7 @@ dependencies = [
 [[package]]
 org = "ballerina"
 name = "constraint"
-version = "1.5.0"
+version = "1.4.0"
```

Comment:
don't we need to update constraint to the latest?

Reply:
This PR only aimed to fix the compiler plugin imprmovement. I will send a separate PR to bump the versions when all the upstream dependencies are updated.